real_check/real_check.ps1

```diff
@@ -0,0 +1,57 @@
+#Copyright (c) Microsoft. All rights reserved.
+#Licensed under the MIT license. See LICENSE file in the project root for full license information.
+function Check-Symbols {
+    param(
+        [string] $lib
+    )   
+        try {
+            [string]$output = cmd.exe /c dumpbin /ALL $lib
+        }
+        catch {
+            Write-Error  "Unable to run dumpbin on given lib"
+            return 2
+        }
+        $start_token = "public symbols"
+        $end_token = "Archive member name"
```

Comment:
Are these tokens documented publicly? Wondering if the binary layout could change at some point and we wouldn't know (or would want some kind of version check).

Reply:
I could not find any public documentation on dumpbin's output format. Might be a good idea to fix dumpbin version.